### ISSUE TITLE
Get Blazor SDK tests working on Helix

### ISSUE THREAD

**Issue by captainsafia** [dotnet/aspnetcore#21547]
PR https://github.com/dotnet/aspnetcore/pull/18962 contains some Blazor build-specific test failures that need to be looked at before we can merge.

Making this issue for housekeeping so this work is tracked.

**Comment by mkArtakMSFT** [comment]
https://github.com/dotnet/aspnetcore/pull/24478

**Comment by mkArtakMSFT** [comment]
Closing this in favor of the #27455
